docs/how-to/release_process/canary.rst

```diff
@@ -0,0 +1,107 @@
+Canary Testing for Checkbox Edge Version: In-depth Process
+==========================================================
+
+Introduction
+------------
+
+The following sections provide a detailed walkthrough of the Canary Testing process for the Checkbox Edge version, starting from snap building to the outcomes of the testing.
+
+.. toctree::
+   :maxdepth: 1
+
+   canary_pipeline
+   validation_job_example
+   validation_pipeline_execution
+   
+
+Building and Publishing the Checkbox Snap
+-----------------------------------------
+
+Trigger Conditions
+^^^^^^^^^^^^^^^^^^
+The GitHub action responsible for building the snap runs every day. However, it only triggers when the following condition is met:
+
+At least one (PR) has been merged since the last edge build.
+
+Build URL
+^^^^^^^^^
+
+To monitor the build process, or to review the configuration and logs, visit the following URL:
+
+https://github.com/canonical/checkbox/actions/workflows/checkbox-snap-daily-builds.yml
+
+Post-Build Actions
+^^^^^^^^^^^^^^^^^^
+
+Once the build is successful:
+
+The snap packages are automatically pushed to the Snap Store.
+These snaps are released to the edge channel, allowing testers and early adopters to access the latest version.
+
+
+Jenkins Monitoring and Validation
+---------------------------------
+
+Snap Monitoring
+^^^^^^^^^^^^^^^
+
+Once the snap is published to the edge channel in the Snap Store, our Jenkins job titled `checkbox-edge-validation-detect-new-build` gets into action.
+
+Monitoring job URL
+^^^^^^^^^^^^^^^^^^
+
+This Jenkins job monitors the Snap Store for the presence of the new snap using the following URL:
+
+[https://api.snapcraft.io/v2/snaps/find?q=checkbox22&channel=edge&fields=revision&architecture=amd64](https://api.snapcraft.io/v2/snaps/find?q=checkbox22&channel=edge&fields=revision&architecture=amd64)
+
+The specific JSON path that's being monitored for changes is:
+
+``$.results[0].revision.revision``
+
+The job checks this path every minute for updates.
+
+Validation and Wait Logic
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Upon detecting a new snap:
+
+1. The job verifies the presence of all other related checkbox snaps.
+2. If any snap is missing, the job waits for an hour, periodically checking for its availability.
+3. If the snaps are available within the waiting period, the next stage of testing is initiated.
+
+checkbox-edge-canary-validation Pipeline
+----------------------------------------
+
+Upon successful snap validation, the ``checkbox-edge-canary-validation`` pipeline begins its operation.
+
+The :doc:`canary_pipeline` contains the groovy script implementing the pipeline.
+
+Testing Platforms and Specifications
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+
+The pipeline concurrently runs 3 distinct jobs on different platforms:
+
+1. **amd64 ubuntu core 22**: Utilizing machines that consume Testflinger tasks from the ``dearest team`` queue.
```

Comment:
Maybe add a little comment on what `dearest team` queue is (you do this below for the RPi4, and it helps)

Reply:
Fixed in ec464efd6